bson/bsoncodec/struct_tag_parser_test.go

```diff
@@ -13,12 +13,13 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
-func TestDefaultStructTagParser(t *testing.T) {
-	testCases := []struct {
+func TestStructTagParsers(t *testing.T) {
```

Comment:
This feels like a hacky way to organize tests. It might be cleaner to to add a field to specify a parser as part of a test case. Then, the test runner loop doesn't need to special-case based on parser name or dynamically switch out the parser.

Reply:
done